sshmenu/sshmenu.py

```diff
@@ -215,14 +216,25 @@ def update_targets():
         targets = config['targets']
 
 
+def get_last_selected_target_index():
+    config = json.loads(resources.user.read(config_name))
+    return config.get('saved_last_chosen_index', 0)
+
+
+def save_last_selected_target_index(index: int):
+    config = json.loads(resources.user.read(config_name))
+    config['saved_last_chosen_index'] = index
+    resources.user.write(config_name, json.dumps(config, indent=4))
+
+
 def display_menu():
-    global targets
+    global targets, config_name
```

Comment:
minor: we don't need the `config_name` variable here anymore 

(side note: I'm pretty sure we can remove this entire line, as `global` is only necessary if you want to modify global variables from a local context, which we are not doing. But I'm okay with leaving it in place anyways)